pkg/controllers/cloud/networkpolicy_controller.go

```diff
@@ -653,6 +666,8 @@ func (r *NetworkPolicyReconciler) backgroupProcess() {
 
 // SetupWithManager sets up NetworkPolicyReconciler with manager.
 func (r *NetworkPolicyReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	configmapController = getConfigMapControllerInstance()
```

Comment:
you dont need to store this in global variable.. Set use GetInstancea and then Register in 1 line

Reply:
Solved.